File: InMotionGIT.Common - Copy/Services/Contracts/DataParameter.vb
Imports System.Runtime.Serialization

Namespace Services.Contracts

    <DataContract()>
    Public Class DataParameter

        <DataMember()>
        Public Property Name As String

        <DataMember()>
        Public Property Type As DbType

        <DataMember()>
        Public Property Size As Integer

        <DataMember()>
        Public Property IsNull As Boolean

        <DataMember()>
        Public Property Value As Object

        <DataMember()>
        Public Property Direction As ParameterDirection

    End Class

End Namespace

File: Common/InMotionGIT.Common.Domain/DataAcceess/DataParameter.cs
using System.Data;
using System.Runtime.Serialization;

namespace InMotionGIT.Common.Domain.DataAccess;


[DataContract()]
public class DataParameter
{

    [DataMember()]
    public string Name { get; set; }

    [DataMember()]
    public DbType Type { get; set; }

    [DataMember()]
    public int Size { get; set; }

    [DataMember()]
    public bool IsNull { get; set; }

    [DataMember()]
    public object Value { get; set; }

    [DataMember()]
    public ParameterDirection Direction { get; set; }

}